### PR TITLE
Task completed: GP-235 Add sound weapon replacement

### PR DESCRIPTION
**Pull request by Kjusok**
https://lighteatersstudio.atlassian.net/jira/software/projects/GP/boards/1?selectedIssue=GP-235

Added class responsible for weapon sounds

### CODE REVIEW

File: Assets/Scripts/Infrastructure/Installers/AudioInstaller.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using Audio;
+using Audio.Gameplay.Weapon;
```
Comment (ega-ega) [inline Assets/Scripts/Infrastructure/Installers/AudioInstaller.cs]: PLs Remove

File: Assets/Prefabs/Audio/WeaponSoundSettings.asset [hunk 2]
```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
```
Comment (ega-ega) [inline Assets/Prefabs/Audio/WeaponSoundSettings.asset]: remove this file

File: Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using Zenject;
+
+namespace Audio.Gameplay.Weapon.Elements
+{
+    public class SoundWeapon : MonoBehaviour
+    {
+        [SerializeField] private ExtendedAudioClip _pickingUp;
+        [SerializeField] private ExtendedAudioClip _shot;
```
Comment (ega-ega) [inline Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs]: remove unused field

File: Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        [SerializeField] private ExtendedAudioClip _reloading;
```
Comment (ega-ega) [inline Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs]: remove unused field

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Prefabs/Audio/WeaponSoundSettings.asset (hunk 2 above)
Comment (Kjusok) [reply Assets/Prefabs/Audio/WeaponSoundSettings.asset]: Done

File: Assets/Scripts/Infrastructure/Installers/AudioInstaller.cs (hunk 1 above)
Comment (Kjusok) [reply Assets/Scripts/Infrastructure/Installers/AudioInstaller.cs]: Done

File: Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs (hunk 3 above)
Comment (Kjusok) [reply Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs]: Done

File: Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs (hunk 4 above)
Comment (Kjusok) [reply Assets/Scripts/Audio/Gameplay/Weapon/Elements/SoundWeapon.cs]: Done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**